node.js/tests/cloudwatchlogs.test.js

```diff
@@ -0,0 +1,62 @@
+// Method:
+const cloudwatchlogs = require('../index');
+
+// Utilities:
+const utils = require('./utils');
+
+// Initialization:
+beforeAll(() => {
+    utils.setEnv('LOGDNA_TAGS', 'jest,unittesting,testing,lambda,logdna,logging,test');
+});
+
+// Test Suites:
+describe('CloudWatchLogs Testing', () => {
+    
+    test('Without Ingestion Key', (done) => {
+        utils.setEnv('LOGDNA_KEY', '');
+        cloudwatchlogs.handler(utils.setData({
+            message: 'This is Sample Log Line for CloudWatch Logging...'
+            , group: 'sampleGroup'
+            , stream: 'testStream'
+            , type: 'cloudwatchlogs'
+        }), {}, (error, result) => {
+            expect(error).toBe('Please, Provide LogDNA Ingestion Key!');
+            expect(result).toBe(undefined);
+            return done();
+        });
+    });
+
+    test('With Dummy Ingestion Key', (done) => {
```

Comment:
This is actually more than a unit test, it's more an integration test. Not sure we want to do this.

Reply:
We are gonna change this once I create `CloudWatchLogs` class and refactor the code.